APIs/openEO/Glossary.qmd

```diff
@@ -47,10 +53,29 @@ In the EO domain, it is common to be implicit about the temporal dimension and j
 Special cases are raster and [vector datacubes](https://r-spatial.org/r/2022/09/12/vdc.html){target="_blank"}.
 Learn more about datacubes in the [datacube documentation](https://openeo.org/documentation/1.0/datacubes.html){target="_blank"}.
 
+## Vector data
+
+In general, **vector data** represent specific things (also called "features") in a space, e.g. on the surface of the Earth.
+
+A **coordinate** represents a specific point in space.
+
+A **feature** is a thing that usually has a geometry (e.g. the outline of an agricultural field, a forest or an urban area) and it may have additional properties assigned (e.g. a name, a color or a population).
+In rare cases features may not have a geometry, which is often referred to as an "empty geometry".
+
+**Geometries** consist of one or more coordinates that may be connected and then form a specific type of geometry, e.g. two points can be connected to a straight line and four straight lines can be connected to rectangle.
+
+Commonly used types of geometries are:
+- Point
+- LineString (connected straight line pieces)
+- Polygon (connected straight line pieces forming a closed ring, possibly with holes - for example a triangle or rectangle)
+
+Multiple geometries of the same type can be combined into a group of geometries, e.g. a so-called "multi point" or  "multi polygon".
```

Comment:
It should be made more clear that a multi-polygon is to be considered to be a single geometry, now it looks like it is a group of geometries, like a feature collection in GeoJSON-speak

Reply:
Rephrase it